src/app/games/lodestone/tests/LodestoneDummyMinimax.spec.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable max-lines-per-function */
+import { Coord } from 'src/app/jscaip/Coord';
+import { Player } from 'src/app/jscaip/Player';
+import { Table } from 'src/app/utils/ArrayUtils';
+import { MGPOptional } from 'src/app/utils/MGPOptional';
+import { LodestoneDummyMinimax } from '../LodestoneDummyMinimax';
+import { LodestonePiece, LodestonePieceLodestone, LodestonePieceNone, LodestonePiecePlayer } from '../LodestonePiece';
+import { LodestoneNode, LodestoneRules } from '../LodestoneRules';
+import { LodestoneLodestones, LodestonePressurePlate, LodestonePressurePlates, LodestoneState } from '../LodestoneState';
+
+describe('LodestoneDummyMinimax', () => {
+    let rules: LodestoneRules;
+    let minimax: LodestoneDummyMinimax;
+
+    const N: LodestonePiece = LodestonePieceNone.UNREACHABLE;
+    const _: LodestonePiece = LodestonePieceNone.EMPTY;
+    const A: LodestonePiece = LodestonePiecePlayer.ZERO;
+    const B: LodestonePiece = LodestonePiecePlayer.ONE;
+
+    beforeEach(() => {
+        rules = LodestoneRules.get();
+        minimax = new LodestoneDummyMinimax(rules, 'LodestoneDummyMinimax');
+    });
+    it('should propose 618 moves at first turn', () => {
+        // Given the initial state
+        const node: LodestoneNode = new LodestoneNode(LodestoneState.getInitialState());
+
+        // When computing all possible moves
+        // Then there should be 618 possible moves
+        // For each empty coord, each lodestone can be placed in 4 different position
+        // For each position, we have to count the possible captures and how they can be placed
+        // The total amounts to 618
+        expect(minimax.getListMoves(node).length).toBe(618);
```

Comment:
quand ya l'explication du nombre en question ça fait sens de mettre ça, mais là avec l'explication très partielle c'est moyen non :D ?

Reply:
donc pas d'explication du tout ? :D J'avais à la base un détail mais 100 lignes de commentaires c'est bofibof, du coup une explication partielle permet de recalculer sans partir de 0

Reply:
Non mais, p-e que la nature d'un assert c'est soit le comportement attendu du code ou du programme, là vu qu'on sais pas trop pourquoi 618, c'est du coup pas notre projet pour le code, c'est juste qu'on lui fait confiance et on vérifie pas manuellement, mais c'est p-e pas hygiénique de mettre un it(expect_code_to_behave_like_it_behave_when_I_coded_it) non ?

Reply:
bah ça a été calculé le 618 mais je sais pas si on veut recalculer le tout ?
Puis oui, ça a du sens même le `it(expect_code_to_behave_like_it_behave_when_I_coded_it)` d'une certaine façon, on veut détecter toute régression, donc vaut mieux ça que laisser passer inaperçu non ?

Reply:
P-e mettre "should propose at least" dans le titre.
Oui pour la non régression mais les titres des tests devraient représenter le besoin réel comportemental :p
Et non on va p-e pas recalculer haha :D quoi qu'y a pire :D

Reply:
changé en at least / greater than !